### PR TITLE
Update list view expansion example

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Demonstrates use of item’s data inside the expansion content.

### CODE REVIEW

File: src/views/listview/examples/list-view.js [hunk 1]
```diff
@@ -521,10 +521,11 @@
            <div class="close">
              <span class="pficon pficon-close" ng-click="$parent.$ctrl.customScope.collapseItem($parent.item)"></span>
            </div>
-           <div ng-if="$parent.item.expandField === 'hosts'" ng-include="'views/listview/examples/hosts-content.html'"></div>
-           <div ng-if="$parent.item.expandField === 'clusters'" ng-include="'views/listview/examples/clusters-content.html'"></div>
-           <div ng-if="$parent.item.expandField === 'nodes'" ng-include="'views/listview/examples/nodes-content.html'"></div>
-           <div ng-if="$parent.item.expandField === 'images'" ng-include="'views/listview/examples/images-content.html'"></div>
+          <span>No if: {{$parent.item.name}}</span>
```
Comment (dtaylor113) [inline src/views/listview/examples/list-view.js]: Do not understand the purpose of "No if: ..." ?

Comment (jeff-phillips-18) [inline src/views/listview/examples/list-view.js]: heh, left over from testing

Comment (jeff-phillips-18) [inline src/views/listview/examples/list-view.js]: removed

**Comment by cdcabrera** [comment]
@jeff-phillips-18 a little late, but sync'd and reviewed, lgtm

**Pull request merged**